### ISSUE TITLE
Will hub.load() models be updated if encoder is finetuned in a MLM task?

### ISSUE THREAD

**Issue by YihanCao123** [tensorflow/models#10473]
Hi, I have a question concerning tf_hub and masked lm models.

In the documentation, it says that a `mlm` task can be trained using:
```
mlm_inputs = dict(
    input_word_ids=tf.keras.layers.Input(shape=(seq_length,), dtype=tf.int32),
    input_mask=tf.keras.layers.Input(shape=(seq_length,), dtype=tf.int32),
    input_type_ids=tf.keras.layers.Input(shape=(seq_length,), dtype=tf.int32),
    masked_lm_positions=tf.keras.layers.Input(shape=(num_predict,), dtype=tf.int32),
)

encoder = hub.load("https://tfhub.dev/tensorflow/bert_en_uncased_L-12_H-768_A-12/4")
mlm = hub.KerasLayer(encoder.mlm, trainable=True)
mlm_outputs = mlm(mlm_inputs)
mlm_logits = mlm_outputs["mlm_logits"]  # [batch_size, num_predict, vocab_size]
```

But what about multitask pretraining using this mlm task? I mean it seems like only the `mlm` weights are updated during finetune, will the encoder weights be updated during this masked lm task? Because I might need the encoder to perform other tasks. If not, 
is there anyway to do a mlm multitask using hub models? Thanks!

**Comment by saberkun** [comment]
Could you check the usage with tf hub team? 
If you do hub.KerasLayer("https://tfhub.dev/tensorflow/bert_en_uncased_L-12_H-768_A-12/4"), will you allow you access the mlm attribute?

**Comment by YihanCao123** [comment]
Thanks, I've found an alternative way to perform the task. Extracting the word embedding layer will work. Thanks for your answer.
